coding-the-legislation/25_vectorial_computing.md

```diff
@@ -20,6 +20,33 @@ array([45, 42, 17])
 
 Basic operations such as `+` or `*` behave the same way on vectors than on numbers, you can thus use them in OpenFisca. However, some operations and structures must be adapted.
 
+## Scalars
+
+As a first-time developer, it is natural to write such a formula
+```py
```

Comment:
I recommend adding a blank line before and after fenced code blocks, as they will render on their own paragraph in any case, for similarity with standard paragraphs that need an empty line to appear as such.